### ISSUE TITLE
Very long `in-line code sequences` do not wrap

### ISSUE THREAD

**Issue by justinsteven** [omnivore-app/omnivore#3382]
I am using https://minimaxir.com/2023/12/chatgpt-structured-data/  as a test case.

The original HTML has things such as:

```
I fed ChatGPT the text of this blog post along with a command to <code>Summarize this blog post into 3 distinct bullet points and a short blurb for sharing on social media, and also provide a list of 3 post categories and a list of 3 SEO keywords:</code>
```

This renders in a browser as:

![image](https://github.com/omnivore-app/omnivore/assets/1893909/abf333f4-7b52-4982-8c43-28b1ce73fced)

Note that the very long content within the `<code>in-line code sequence</code>` is wrapped by the browser.

Once ingested into Omnivore, it renders as follows on the web:

![image](https://github.com/omnivore-app/omnivore/assets/1893909/5928d118-2a58-4c45-918d-e3c0ef77bbac)

i.e. it lacks wrapping, requiring the user to scroll horizontally

It looks similar on the iOS and iPadOS apps

As a hack, removing the `white-space: pre` styling using the browser console causes the in-line code to be wrapped. I think this is much more readable. I am not sure that removing this particular styling rule is a correct or sensible fix though.

![image](https://github.com/omnivore-app/omnivore/assets/1893909/a8bbcc15-ed1f-4f23-a210-6f1ceedc264d)

Suggested fix: Always wrap very long `<code>in-line code sequences</code>` on the Web UI and on mobile apps.

**Comment by justinsteven** [comment]
This duplicates #2658 but IMO has more detail

**Comment by justinsteven** [comment]
Looks like this was fixed in #3359 (perhaps by accident, I know @jacksonh was locally hacking on this when I brought it up on Discord and it might have accidentally leaked in to a commit). If this change sticks, this issue and #2658 can be closed
